.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: DotNet Core org issues
```

Spelling/branding tweak:

```suggestion
  - name: .NET issues
```